tests/testthat/_snaps/assert.md

```diff
@@ -12,7 +12,7 @@
     Code
       vec_assert(lgl(), factor())
     Condition
-      Error in `vec_assert()`:
+      Error:
```

Comment:
So the error is considered to come from the "top level" now, so that is why we don't get the function name in the error?

Reply:
Yes, in this case the error call is actually `eval()` from the testthat REPL, and this is hidden by rlang.